app/util/autoversioning.py

```diff
@@ -144,11 +82,13 @@ def _repo_has_uncommited_changes():
 
 def _is_commit_hash_in_masters_first_parent_chain(commit_hash):
     """
-    Check if the current HEAD is in the first-parent chain of origin/master. The first-parent chain of origin/master
-    consists of all the "trunk" commits. All other commits are either on merged branches or haven't been merged at all.
+    Check if the current HEAD is in the first-parent chain of origin/master. The first-parent chain
+    of origin/master consists of all the "trunk" commits. All other commits are either on merged
+    branches or haven't been merged at all.
 
     :type commit_hash: str
     :rtype: bool
+    :raises CalledProcessError: if git not a git repo or is a shallow clone
```

Comment:
Should say `if app is not in a git repo`

Reply:
ayyyyy. Hi @nickzuber !